test/services/sync/repo/school.repo.integration.test.js

```diff
@@ -32,50 +32,72 @@ describe('school repo', () => {
 	after(async () => {
 		await server.close();
 	});
+	describe('createSchool', () => {
+		it('should successfully create new school', async () => {
+			const schoolName = 'Test School';
+			const currentYear = new ObjectId();
+			const federalStateId = new ObjectId();
+			const systemId = new ObjectId();
+			const ldapSchoolIDn = 'TEST_LDAP';
+			const fileStorageType = 'awsS3';
+			const school = {
+				name: schoolName,
+				systems: [systemId],
+				ldapSchoolIdentifier: ldapSchoolIDn,
+				currentYear,
+				federalState: federalStateId,
+				fileStorageType,
+			};
+			const res = await SchoolRepo.createSchool(school);
+			createdSchools.push(res);
+			expect(res._id).to.be.not.undefined;
```

Comment:
.eq() you can match a complete object for the test. It is more readable like many expects